src/libs/response.js

```diff
@@ -71,39 +49,46 @@ class Response {
-		}
-		return content
-	}
 
 	/**
-	 *
 	 * Sending response
 	 * @param {String} [content=``] Main content of the message to be displayed.
-	 * @param {Object} [plugins={}] List of plugins to be applied into the message.
+	 * @param {Object} plugins List of plugins to be applied into the message.
+	 * @param {Array} plugins.socket optional message modifiers.
+	 * @param {String} plugins.color for the embed color. [Hex code]
+	 * @param {String} plugins.url as the url for an embed.
+	 * @param {Buffer} plugins.image as the attachment url.
+	 * @param {String} plugins.imageGif as the attachment url.
+	 * @param {String} plugins.thumbnail as embed icon.
+	 * @param {String} plugins.header use header in an embed.
+	 * @param {String} plugins.footer use footer in an embed.
+	 * @param {String} plugins.customHeader First index as header text and second index as header icon.
+	 * @param {String} plugins.timestamp Toggle true to include the timestamp on the message.
+	 * @param {String} plugins.status Set to `sucesss`, `warn`, or `fail` to indicate the status of the message.
+	 * @param {String} plugins.topNotch Adds topnotch with custom message. Leave blank/null to ommit it.
+	 * @param {Boolean} plugins.simplified as non embed message toggle.
+	 * @param {Boolean} plugins.notch as huge blank space on top and bottom
+	 * @param {Boolean} plugins.prebuffer as indicator if parameter supply in "image" already contains image buffer.
+	 * @param {Boolean} plugins.paging Toggle true to use paging mode in the message.
+	 * @param {Boolean} plugins.cardPreviews Toggle true to allow the embed to display the card.
+	 * @param {Boolean} plugins.raw Toggle `true` to return the message's composition without sending it to the target field.
+	 * @param {Boolean} plugins.timestampAsFooter Toggle `true` to include the message timestamp in the footer of embed.
+	 * @param {Boolean} plugins.directMessage Indicate if the message is a DM
+	 * @param {Boolean} plugins.feedback  beta feature, **not used often**
+	 * @param {Boolean} plugins.fetchReply Application command option to grab reference
+	 * @param {Boolean} plugins.ephemeral Application command option to hide message from public
+	 * @param {Object | String} plugins.field message field target (GuildChannel/DM).
+	 * @param {String | Number} plugins.deleteIn as countdown before the message get deleted. In seconds.
+	 * @param {Array | String | Object} plugins.components Array of components like buttons
+	 * @param {Object} plugins.file	object for local file attachments
+	 * @param {String} plugins.file.filePath local file file path
+	 * @param {String} plugins.file.fileName local file name
+	 * @param {String} plugins.file.fileDescription local file description
 	 * @return {void}
 	 */
 	async send(content = ``, plugins = {}) {
-	
 		let socket = plugins.socket || []
 		let color = plugins.color || palette.crimson
-		plugins.color = color
```

Comment:
@Moore2021 pan are you really sure removing this line? i believe ~~some years ago~~ this was for patch fix the default embed color / profile card's main accent

Reply:
I don't know why it was there prior; initial testing hasn't found any issues but if you can remember where the error occurred and we can reproduce I'll gladly reconfigure that.